dangerfile.js

```diff
@@ -28,16 +28,12 @@ import {
   propKeys as iosPropKeys,
 } from './packages/bpk-tokens/tokens/base.raw.ios.json';
 
-const BACKPACK_SQUAD_MEMBERS = [
-  'georgegillams',
-  'jamesf3rguson',
-  'k0nserv',
-  'matteo-hertel',
-  'matthewdavidson',
-  'shaundon',
-  // TODO Will needs a GitHub account.
-  // TODO Use the API for this. It requires auth though so this is ok for now.
-];
+import * as meta from './meta.json';
+
+const BACKPACK_SQUAD_MEMBERS = meta.maintainers.map(
+  maintainer => maintainer.github,
+);
+console.log(BACKPACK_SQUAD_MEMBERS);
```

Comment:
Remove this

Reply:
Dw I already have locally :) I was just using it to test on CI